jax/experimental/jax2tf/jax2tf.py

```diff
@@ -539,7 +551,12 @@ def process_call(self, call_primitive: core.Primitive, f: lu.WrappedFun,
     assert call_primitive.multiple_results
     vals: Sequence[TfVal] = [t.val for t in tracers]
     f = _interpret_subtrace(f, self.main, tuple(t.aval for t in tracers))
-    vals_out: Sequence[Tuple[TfVal, core.AbstractValue]] = f.call_wrapped(*vals)
+    if call_primitive.name == "named_call":
```

Comment:
It feels bad to anchor on the string name of a primitive. For named_call in particular I think there are plans to upstream the implementation to JAX core (there are equivalent implementations in Haiku and Flax). I suggest we do that first and then your test can be `if call_primitive == lax.named_call_p`.

cc @LenaMartens

Reply:
I'm happy to wait for the upstream the implementation to JAX core. Is there already a PR or I'm also happy to help with the upstreaming. @levskaya @gnecula

Reply:
Upstreaming is tracked as issue #4558. I had assigned it to myself, but at the moment anyone is welcome to take it if they'd like.

Reply:
PR #4733 has merged, changed to use if call_primitive == lax.named_call_p.